Unix/provmgr/provmgr.c

```diff
@@ -2052,7 +2115,6 @@ MI_Result ProvMgr_Init(
     self->timeoutHandler.sock = INVALID_SOCK;
     self->timeoutHandler.data = self;
     self->timeoutHandler.callback = _TimeoutCallback;
-    self->timeoutHandler.handlerName = MI_T("PROVMGR_TIMEOUT_MANAGER");
```

Comment:
I did do a rebase.  Actually, I created a new branch so that I could cherry pick these changes.  I missed this change when I copied it over.